src/index.html

```diff
@@ -1,13 +1,378 @@
+          </li>
+          <li class="menu__item">
+            <a class="menu__link" href="#get-in-touch">Get in touch</a>
+          </li>
+
+          <div class="menu__lang-button">
+            <div class="header__nav-languages menu__languages">
+              <a href="#" class="header__nav-func_item header__check-lang--en">en</a>
+              <a href="#" class="header__nav-func_item header__check-lang--ua">ua</a>
+            </div>
+
+            <a href="#" class="header__nav-func_item button button--buy">buy</a>
+          </div>
+        </ul>
+
+      </nav>
+
+      <a href="#" class="scroll-up"></a>
+    </header>
+
+    <main class="main">
+      <section class="main__benefits benefits">
+        <section class="benefits__item">
+          <div class="benefits__icon-design benefits__icon">
+          </div>
+
+          <h2 class="benefits__headline">Futuristic Design</h2>
+          <p class="benefits__paragraph main__paragraph">
+            To give Luna a truly flawless look, we specifically
+            picked aircraft grade aluminum as its material and
+            adopted both three-dimensional stretch-bending
+            technology and a high precision cold forging
+            technique.
+          </p>
+        </section>
+
+        <section class="benefits__item">
+          <div class="benefits__icon-speaker benefits__icon">
+          </div>
+
+          <h2 class="benefits__headline">Tweeter Speaker System</h2>
+          <p class="benefits__paragraph main__paragraph">
+            To deliver a more layered sound performance
+            better than a sole full-range speaker, our team
+            equipped Luna with one more tweeter speaker
+            responsible for high-frequency sound independently.
+          </p>
+        </section>
+
+        <section class="benefits__item">
+          <div class="benefits__icon-multiroom benefits__icon">
+          </div>
+
+          <h2 class="benefits__headline">Multiroom System</h2>
+          <p class="benefits__paragraph main__paragraph">
+            Luna is natively compatible with your home Wi-Fi.
+            Set up multiple speakers in different rooms to
+            expand your music experience into the entire
+            house.
+          </p>
+        </section>
+
+        <section class="benefits__item">
+          <div class="benefits__icon-lighting benefits__icon">
+          </div>
+
+          <h2 class="benefits__headline">Intuitive Lighting System</h2>
+          <p class="benefits__paragraph main__paragraph">
+            An intuitive user interface allows you to adjust the
+            hue and saturation of color for lighting that fits any
+            mood and situation.
+          </p>
+        </section>
+      </section>
+
+      <div class="main__presentation presentation">
+        <img
+          class="presentation__pctr"
+          src="images/presentation.png"
+          alt="Meet Luna">
```

Comment:
Check your code style 
```suggestion
        <img
          class="presentation__pctr"
          src="images/presentation.png"
          alt="Meet Luna"
        >
```

Reply:
✔️